app/uk/gov/hmrc/tradergoodsprofilesdatastore/controllers/EmailController.scala

```diff
@@ -35,11 +35,12 @@ class EmailController @Inject() (
     with Logging {
 
   def getEmail(eori: String): Action[AnyContent] = identify.async { implicit request =>
-    customsDataStoreConnector
-      .getEmail(eori)
-      .map {
-        case Some(email) => Ok(Json.toJson(email))
-        case None        => NotFound
-      }
+    for {
```

Comment:
for comprehension is only needed when there are multiple flatmaps - if it is one flatmap or less i would leave it